src/Console/Command/FixCommand.php

```diff
@@ -105,7 +106,8 @@ protected function configure()
                     new InputOption('diff', '', InputOption::VALUE_NONE, 'Also produce diff for each file.'),
                     new InputOption('format', '', InputOption::VALUE_REQUIRED, 'To output results in other formats.'),
                     new InputOption('stop-on-violation', '', InputOption::VALUE_NONE, 'Stop execution on first violation.'),
-                    new InputOption('show-progress', '', InputOption::VALUE_REQUIRED, 'Type of progress indicator (none, run-in, or estimating).'),
+                    new InputOption('show-progress', '', InputOption::VALUE_REQUIRED, 'Type of progress indicator (none, run-in or estimating).'),
+                    new InputOption('columns', '', InputOption::VALUE_REQUIRED, 'Number of columns to use for progress output (integer or "max").'),
```

Comment:
let us have
- none
- run-in
- estimating
- estimating-max

and remove `run-in` on 3.x line